autorest/jsonrpc/__init__.py

```diff
@@ -120,7 +120,7 @@ def message(self, channel: Channel, text: str) -> None:
         """Send a log message to autorest.
         """
 
-    def get_boolean_value(self, key: str, default: bool = None) -> bool:
+    def get_boolean_value(self, key: str, default: bool = None) -> Optional[bool]:
```

Comment:
That's an interesting discovery, I really want this guy to return a bool, and not an optional bool, it probably means we need to raise a KeyError if we don't find the key and no default is provided (like a regular dict does)